src/test/java/TestResources/DummyCore.java

```diff
@@ -21,29 +21,48 @@
 
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
+import org.darisadesigns.polyglotlina.Desktop.CustomControls.DesktopInfoBox;
+import org.darisadesigns.polyglotlina.Desktop.DesktopHelpHandler;
+import org.darisadesigns.polyglotlina.Desktop.DesktopIOHandler;
+import org.darisadesigns.polyglotlina.Desktop.DesktopOSHandler;
+import org.darisadesigns.polyglotlina.Desktop.PFontHandler;
+import org.darisadesigns.polyglotlina.Desktop.DesktopPropertiesManager;
 import org.darisadesigns.polyglotlina.DictCore;
-import org.darisadesigns.polyglotlina.PGTUtil;
-import org.darisadesigns.polyglotlina.PolyGlot;
+import org.darisadesigns.polyglotlina.InfoBox;
+import org.darisadesigns.polyglotlina.Desktop.PGTUtil;
+import org.darisadesigns.polyglotlina.Desktop.PolyGlot;
+import org.darisadesigns.polyglotlina.OSHandler;
 
 /**
  * Allows for testing which requires a DictCore without exposing PolyGlot object
  * @author draque
  */
 public class DummyCore extends DictCore {
-    private DummyCore (PolyGlot polyGlot) {
-        super(polyGlot);
+    private DummyCore (DesktopPropertiesManager propsManager, OSHandler osHandler) {
+        super(propsManager, osHandler, new PGTUtil());
     }
     
     public static DummyCore newCore() {
         try {
-            Constructor constructor = PolyGlot.class.getDeclaredConstructor(new Class[]{String.class});
+            InfoBox infoBox = new DesktopInfoBox(null);
+            DesktopHelpHandler helpHandler = new DesktopHelpHandler();
+            PFontHandler fontHandler = new PFontHandler();
+            DesktopOSHandler osHandler = new DesktopOSHandler(DesktopIOHandler.getInstance(), infoBox, helpHandler, fontHandler);
+            Constructor constructor = PolyGlot.class.getDeclaredConstructor(new Class[]{String.class, DictCore.class, DesktopOSHandler.class});
             constructor.setAccessible(true);
-            PolyGlot polyGlot = (PolyGlot)constructor.newInstance(PGTUtil.TESTRESOURCES);
+            DesktopPropertiesManager propsManager = new DesktopPropertiesManager();
+            DummyCore core = new DummyCore(propsManager, osHandler);
+            // Is this now really needed to be constructed?
```

Comment:
This construction method comes from ancient code that I'm going to eventually clean up. It doesn't honestly serve any function any longer except making PolyGlot compile/function when simpler logic would do just as well.